cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/CloudNetWrapperConfig.java

```diff
@@ -181,14 +173,10 @@ public CloudNetWrapperConfig load()
                 save();
             }
 
-            //generatated configurations
-
             this.proxy_config_host = configuration.getString("general.proxy-config-host");
 
             if (!Files.exists(Paths.get(devServicePath)))
```

Comment:
```suggestion
            if (!Files.exists(Paths.get(devServicePath))) {
                Files.createDirectories(Paths.get(devServicePath));
            }
```

Please keep braces after if-statements